aep/general/0148/aep.md.j2

```diff
@@ -1,5 +1,155 @@
 # Standard fields
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/28) for more
-information.
+Certain concepts are common throughout any corpus of APIs. In these situations,
+it is useful to have a standard field name and behavior that is used
+consistently to communicate that concept.
+
+## Guidance
+
+Standard fields **should** be used to describe their corresponding concept, and
+**should not** be used for any other purpose.
+
+### Resource paths and IDs
+
+#### path
+
+Every resource **must** have a `string path` field, used for the resource path
+(AEP-122), which **should** be the first field in the resource.
```

Comment:
```suggestion
Every resource **must** have a `string path` field, used for the [resource path](/resource-paths), which **should** be the first field in the resource.
```

I think it's best if we refer to AEPs via their slugs?

Reply:
I agree, but right now:

1. We use numbers throughout the site.  I think it's better to stay consistent, and then fix them all at once when we're ready to do so.  This is related to...
2. The site generator linkifies AEP-NUM but not AEP-SLUG.